src/main/java/com/ericsson/ei/erqueryservice/PropertyNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.ericsson.ei.erqueryservice;
+
+public class PropertyNotFoundException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    public PropertyNotFoundException(String string) {
+        super(string);
+    }
+}
```

Comment:
Should we have moved this to its correct place?

Reply:
I moved it to the exception package.